protocols/luci-proto-wireguard/luasrc/model/cbi/admin_network/proto_wireguard.lua

```diff
@@ -81,10 +81,22 @@ preshared_key = section:taboption(
             "cryptography for post-quantum resistance.")
 )
 preshared_key.password = true
-preshared_key.datatype = "and(base64,rangelength(44, 44))"
+preshared_key.datatype = "and(base64,rangelength(44,44))"
 preshared_key.optional = true
 
 
+fwmark = section:taboption(
+  "advanced",
+  Value,
+  "fwmark",
+  translate("Firewall Mark"),
+  translate("Optional. 32-bit hexadecimal mark for outgoing packets. " ..
+            "Enter value without leading <code>0x</code>.")
```

Comment:
That's fine. Just keep it as a string if you need... If it starts with `0x` and subsequently you have `[a-zA-Z0-9]{1,8}`, then it's a 32-bit hex number. If it doesn't start with 0x, then it must be ≤4294967296.